pom.xml

```diff
@@ -73,7 +73,12 @@
 	</repositories>
 
 	<dependencies>
-
+		<dependency>
+			<groupId>com.microsoft.azure.functions</groupId>
+			<artifactId>azure-functions-java-core-library</artifactId>
+			<version>1.0.0-SNAPSHOT</version>
```

Comment:
Let's release the core library first then we can add the dependence to the release